examples/7_get_started_with_real_robot.md

```diff
@@ -0,0 +1,782 @@
+    <p style="text-align:center;">1. Zero position</p>
+  </div>
+  <div style="width:30%; margin:10px;">
+    <img src="../media/koch/leader_rotated.webp?raw=true" alt="Koch v1.1 leader arm rotated position" title="Koch v1.1 leader arm rotated position" style="width:100%;">
+    <p style="text-align:center;">2. Rotated position</p>
+  </div>
+  <div style="width:30%; margin:10px;">
+    <img src="../media/koch/leader_rest.webp?raw=true" alt="Koch v1.1 leader arm rest position" title="Koch v1.1 leader arm rest position" style="width:100%;">
+    <p style="text-align:center;">3. Rest position</p>
+  </div>
+</div>
+
+
+Run this code to calibrate and connect your robot:
+```python
+robot.connect()
+
+>>> TODO
+```
+
+Now we will see how to read the position of the leader and follower arms using `read` from [`DynamixelMotorsBus`](../lerobot/common/robot_devices/motors/dynamixel.py). If the calibration is done well, the posiiton should be the similar when both arms are in a similar position.
+
+Run this code to get the positions in degree:
+```python
+leader_pos = robot.leader_arms["main"].read("Present_Position")
+print(leader_pos)
+follower_pos = robot.follower_arms["main"].read("Present_Position")
+print(follower_pos)
+>>> array([-0.43945312, 133.94531, 179.82422, -18.984375, -1.9335938, 34.541016], dtype=float32)
+>>> array([-1.0546875, 128.67188, 174.90234, -7.998047, -5.4492188, 32.34375], dtype=float32)
+```
+
+Importantly, we also converted the "step" position to degree. This is much easier to interpet and debug. In particular, the zero position used during calibration now corresponds to 0 degree for each motor. Also, the rotated position corresponds to 90 degree for each motor.
+
+**Teleoperate**
+
+Now you can easily teleoperate your robot by reading the positions from the leader arm and sending them as goal positions to the follower arm.
+
+Run this code to teleoperate:
+```python
+import tqdm
+# Teleoperate for 30 seconds
+# Fastest communication is done at a frequency of ~200Hz
+for _ in tqdm.tqdm(range(30*200)):
+    leader_pos = robot.leader_arms["main"].read("Present_Position")
+    robot.follower_arms["main"].write("Goal_Position", leader_pos)
+```
+
+You can also teleoperate by using `teleop_step` from [`KochRobot`](../lerobot/common/robot_devices/robots/koch.py).
+
+Run this code to teleoperate:
+```python
+import tqdm
+for _ in tqdm.tqdm(range(30*200)):
+    robot.teleop_step()
+```
+
+Teleoperation is useful to record data. To this end, you can use `teleop_step` from [`KochRobot`](../lerobot/common/robot_devices/robots/koch.py) with `record_data=True`. It outputs the follower position as `"observation.state"` and the leader position as `"action"`. It also converts the numpy arrays into torch tensors, and concatenates the positions in the case of two leader and two follower arms like in Aloha.
+
+Run this code to try data recording during a teleoperation step:
+```python
```

Comment:
not meant for a for-loop because of the prints. I added this:
```
Run this code several time to see how (slowly) moving the leader arm affects the observation and action:
```